Sources/Build/BuildPlan.swift

```diff
@@ -174,6 +174,17 @@ public enum TargetBuildDescription {
         }
     }
 
+    /// The resources in this target.
+    var resources: [Resource] {
+        switch self {
+        case .swift(let target):
+            return target.resources
+        case .clang(let target):
+            // TODO: Clang targets should support generated resources in the future.
+            return target.target.underlyingTarget.resources
```

Comment:
wonder if we can expose that at the target level so we dont have to drill down so deep?

Reply:
I think so, but I'll leave Clang target untouched for this PR, we can clean it up once we are adding support for Clang targets.

Reply:
That makes sense.  When we get to them though, I think it would make sense to have a wrapper as for Swift targets (should that perhaps be added already now, except that for Clang targets it just returns the non-generated ones?).